src/components/card/card.stories.js

```diff
@@ -115,7 +115,11 @@ function makeStory(name, themeSelector) {
   const component = () => {
     const knobs = cardKnobs();
 
-    return ([getCard(knobs), getSmallCard(knobs)]);
+    return (
+      <>
```

Comment:
Same as my earlier comment about empty elements.

Reply:
This has reappeared too.

Reply:
fixed in d1a866b